dev/src/components/CourseUtilityLinks.tsx

```diff
@@ -39,11 +39,11 @@ const CourseUtilityLinks = (): JSX.Element => {
                     Course Info
                 </a>
 
-                {enrollment.hasGroup() ?
+                {state.hasGroup(courseID) ?
                     <Link to={"/course/" + courseID + "/group"} className="list-group-item list-group-item-action">
                         View Group
                     </Link>
-                    : <Link to={"/course/" + courseID + "/group/create"} className="list-group-item list-group-item-action list-group-item-success">
+                    : <Link to={"/course/" + courseID + "/group"} className="list-group-item list-group-item-action list-group-item-success">
```

Comment:
No `/group/create` anymore?

Reply:
The `/group` route now renders, for the given course:
1. Teacher group view if the user is a teacher
2. Group creation view if a student is not in a group
3. Student group view if a student is in a group